src/api/externalUser.ts

```diff
@@ -251,6 +262,7 @@ export class ExternalUser {
             const payload: LoginRequestsMessagePayload = {
                 requests: [loginRequest],
```

Comment:
here i am just thinking do we need to add the loginRequest and datasharingRequest? i am confused with this part

Reply:
yes, we probably want `requests: [loginRequest, dataSharingRequest]`